### DISCUSSION TITLE
How do I flush the database?

### DISCUSSION THREAD

**Discussion by afidegnum** [devashishdxt/rexie#17] [Q&A] [answered]
HI, 

I just discovered your crate, I'm immensely grateful about it. I have been struggling with web-sys idbFactory. 

A quick question: what's the function needed to flush the database to erase everything from it ? 

Cheers,

**Comment by devashishdxt** [comment, marked as answer]
You can use [`Rexie::delete`](https://docs.rs/rexie/0.4.2/rexie/struct.Rexie.html#method.delete)
